test/e2e/user_workload_monitoring_test.go

```diff
@@ -436,6 +380,7 @@ func assertUserWorkloadMetrics(t *testing.T) {
 	err = framework.Poll(5*time.Second, 5*time.Minute, func() error {
 		body, err := f.AlertmanagerClient.GetAlertmanagerAlerts(
 			"filter", `alertname="VersionAlert"`,
+			"filter", fmt.Sprintf(`namespace="%s"`, userWorkloadTestNs),
```

Comment:
we now create tow namespaces, see setupUserApplications
This also make the test better isolated and less error prone